src/afl-fuzz-one.c

```diff
@@ -384,17 +384,17 @@ u8 fuzz_one_original(afl_state_t *afl) {
 
 #else
 
-  if (unlikely(afl->mutator) && unlikely(afl->mutator->afl_custom_queue_get)) {
+  if (unlikely(afl->number_of_custom_mutators )) {
 
     /* The custom mutator will decide to skip this test case or not. */
 
-    if (!afl->mutator->afl_custom_queue_get(afl->mutator->data,
-                                            afl->queue_cur->fname)) {
+    LIST_FOREACH(&afl->custom_mutator_list, struct custom_mutator, {
 
-      return 1;
-
-    }
+      if (el->afl_custom_queue_get && !el->afl_custom_queue_get(el->data, afl->queue_cur->fname)) {
```

Comment:
Does it ever make sense to specify multiple custom queues? I'm not 100% sure how they are supposed to work.

Reply:
the function type decides if a fuzzing of the queue entry should be performed or not. if any of them would say no it should be skipped.

Reply:
Ah I see, thx. By the name of the function I thought the mutator would implement its own queue...
It's more like a selector/filter function then. In that case, stacking them make sense, also.

Reply:
docs/custom_mutators.md ;)